01_login.php

```diff
@@ -54,134 +55,132 @@
 
 		<!-- Navbar (sit on top) -->
 		<div class="w3-top">
-			<div class="w3-bar w3-white w3-wide w3-padding w3-card">
+			<div class="w3-bar w3-grey w3-padding w3-card">
 				<img src="images/placeholder.jpg" width="40" height="40">
-				<a class="w3-bar-item"><b>logo</b> </a>
+				<a class="w3-bar-item"><b>Soundwave</b> </a>
+			</div>
+		</div>
 
-			</div></div>
 
+		<!-- background colour -->
+		<div class="w3-black">
```

Comment:
added a black background container within login page; to change the background from being plain white to black